CHANGELOG.md

```diff
@@ -185,6 +185,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (x/auth) [\#10880](https://github.com/cosmos/cosmos-sdk/pull/10880) Added a new query to the tx query service that returns a block with transactions fully decoded.
 * (types) [\#11200](https://github.com/cosmos/cosmos-sdk/pull/11200) Added `Min()` and `Max()` operations on sdk.Coins.
 * (gov) [\#11287](https://github.com/cosmos/cosmos-sdk/pull/11287) Fix error message when no flags are provided while executing `submit-legacy-proposal` transaction.
+* (ante) [#11413](https://github.com/cosmos/cosmos-sdk/pull/11413) Refactor tx middleware to support pluggable feemarket module.
```

Comment:
we need a lot more here. This is heavily breaking and its unclear if users will adopt it. Can we revert the removal for the old mempoolfee middleware and introduce a new file feemarket.go.

Reply:
This is needed to enable dynamic fee markets. Most of the users won't need it. But those with heavy load will need it. And since there will be many changes in 0.47 - it will take lot of time for other chains to migrate... That's why we want to include this PR in 0.46 - it extends the core proto and will give a time for tools and ecosystem to already plan the support.

Reply:
huh? I don't get how this works with 0.47, I was saying this changelog entry needs to be a lot bigger, there is no mention of what the new api is compared to the old one, also its unclear why we need to break the old api.

Reply:
oh, I was answering to the _unclear if users will adopt it._ and the general review comment _we should not sneak in such a large breaking change right before the release of 0.46-beta_ 
 
Sorry if it created a confusion. BTW, It's not that big change.

Reply:
> there is no mention of what the new api is compared to the old one,

I moved the changelog to API breaking section, and add the actual breakages.

> also its unclear why we need to break the old API.

It's mainly to avoid duplicated executions, because for feemarket implementation, the three steps could be done in a single pass:
- check the specified fee is enough
- decide tx priority
- decide the effective fee (which could be different than the one specified in tx) and deduct it from the sender.

Reply:
all good thanks